NAS-104321: Update ACL management text: (#1663)
freenas/freenas-docs: userguide/storage.rst

ACLs are modified by adding or removing Access Control Entries (ACEs) in
:menuselection:`Storage --> Pools`.
Find the desired dataset, click |ui-options|, and select
:guilabel:`Edit ACL`. The :guilabel:`ACL Manager` opens. The ACL manager

> is required to modify permissions on a dataset that has an ACL.

Reviewer: Ambiguous "required".
```suggestion
must be used to modify permissions on a dataset with an ACL.
```
